Parsers/ASimAuthentication/Tests/Illumio_Core_Authentication_SchemaTest.csv

```diff
@@ -0,0 +1,108 @@
+"(2) Info: Missing optional field [EventOriginalType]"
+"(2) Info: Missing optional field [EventOwner]"
+"(2) Info: Missing optional field [EventProductVersion]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [EventSubType]"
+"(2) Info: Missing optional field [HttpUserAgent]"
+"(2) Info: Missing optional field [LogonMethod]"
+"(2) Info: Missing optional field [LogonProtocol]"
+"(2) Info: Missing optional field [LogonTarget]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDomain]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcOs]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcHostname]"
+"(2) Info: Missing optional field [SrcIsp]"
+"(2) Info: Missing optional field [SrcOriginalRiskLevel]"
+"(2) Info: Missing optional field [SrcPortNumber]"
+"(2) Info: Missing optional field [SrcRiskLevel]"
+"(2) Info: Missing optional field [TargetAppId]"
+"(2) Info: Missing optional field [TargetAppName]"
+"(2) Info: Missing optional field [TargetDescription]"
+"(2) Info: Missing optional field [TargetDeviceType]"
+"(2) Info: Missing optional field [TargetDvcId]"
+"(2) Info: Missing optional field [TargetDvcOs]"
+"(2) Info: Missing optional field [TargetDvcScopeId]"
+"(2) Info: Missing optional field [TargetDvcScope]"
+"(2) Info: Missing optional field [TargetFQDN]"
+"(2) Info: Missing optional field [TargetGeoCity]"
+"(2) Info: Missing optional field [TargetGeoCountry]"
+"(2) Info: Missing optional field [TargetGeoLatitude]"
+"(2) Info: Missing optional field [TargetGeoLongitude]"
+"(2) Info: Missing optional field [TargetGeoRegion]"
+"(2) Info: Missing optional field [TargetIpAddr]"
+"(2) Info: Missing optional field [TargetOriginalAppType]"
+"(2) Info: Missing optional field [TargetOriginalRiskLevel]"
+"(2) Info: Missing optional field [TargetOriginalUserType]"
+"(2) Info: Missing optional field [TargetPortNumber]"
+"(2) Info: Missing optional field [TargetRiskLevel]"
+"(2) Info: Missing optional field [TargetSessionId]"
+"(2) Info: Missing optional field [TargetUrl]"
+"(2) Info: Missing optional field [TargetUserId]"
+"(2) Info: Missing optional field [TargetUserScopeId]"
+"(2) Info: Missing optional field [TargetUserScope]"
+"(2) Info: Missing optional field [TargetUserType]"
+"(2) Info: Missing optional field [TargetUsername]"
```

Comment:
![image](https://github.com/user-attachments/assets/2bb20fe6-6f8e-4601-91b8-5f2038b34b57)

Reply:
Here, TargetUserName and TargetUserNameType are the relevant fields I am including with an alias to User as well.